### PR TITLE
Adding the DATS model dataset descriptor file to the visual-working-memory dataset

### PR DESCRIPTION
**Pull request by gi114**
PR to add the DATS model dataset descriptor to the visual-working-memory dataset. Please double check that the base branch to merge the dats-descriptor branch to is correctly selected as master

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (PapillonMcGill) [inline .idea/misc.xml]: files in .idea directory not pertinent for the dataset.

File: dats.json [hunk 2]
```diff
@@ -0,0 +1,151 @@
+{
+    "$schema": "https://w3id.org/dats/schema/dataset_schema.json",
```
Comment (PapillonMcGill) [inline dats.json]: all lines starting with "$schema:" need to be remove.

You can validate the DATS file using https://github.com/conpdatasets/1000GenomesProject/blob/44a7ee921ec5e1ced3991fab443d50f5af94d2bb/scripts/1000genomes_validate_json.pl

Comment (gi114) [reply dats.json]: Hi @PapillonMcGill, thank you for your comment. I used the perl script to validate the dats file but it does not recognize some of the fields that should be "required" in dats. I will remove $schema lines.
What is your name on slack? I think we should meet to have a proper discussion about DATS fields and their validation. Let me know

**Comment by gi114** [comment]
Hi, I updated this PR with a modified version of the dats.json

File: .idea/misc.xml (hunk 1 above)
Comment (PapillonMcGill) [reply .idea/misc.xml]: @gi114 Please remove all files in .idea directory from this PR.

**Comment by gi114** [comment]
Hi @PapillonMcGill, would you let me know on this PR ?

**Comment by PapillonMcGill** [comment]
@gi114 Please remove the .idea directory and all files in it from this PR. 
You can do a `git rm --cache` to keep them on your disk while removing from git

**Pull request merged**